src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -604,26 +604,30 @@ object DatabaseConversions {
     }
   }
 
-  implicit val bakingRightsToRows = new Conversion[Id, (BlockHash, BakingRights), Tables.BakingRightsRow] {
-    override def convert(from: (BlockHash, BakingRights)): tezos.Tables.BakingRightsRow = {
-      val (blockHash, bakingRights) = from
+  implicit val bakingRightsToRows = new Conversion[Id, ((Option[Int], Option[Int], BlockHash), BakingRights), Tables.BakingRightsRow] {
+    override def convert(from: ((Option[Int], Option[Int], BlockHash), BakingRights)): tezos.Tables.BakingRightsRow = {
+      val (blockHashWithCycleAndGovernancePeriod, bakingRights) = from
```

Comment:
I suggest getting rid of `_1`,`_2` etc with something like that:
```
      val ((cycle, governancePeriod, blockHash), bakingRights) = from
      bakingRights
        .into[Tables.BakingRightsRow]
        .withFieldConst(_.blockHash, blockHash.value)
        .withFieldConst(_.estimatedTime, toSql(bakingRights.estimated_time))
        .withFieldConst(_.cycle, cycle)
        .withFieldConst(_.governancePeriod, governancePeriod)
        .transform
```

Reply:
Duh! I agree. :)